### PR TITLE
Minor Corrections to Help Manual

### PR DESCRIPTION
**Pull request by DaveC49**
Fix on author entry. 
Correct external reference and link to gnucash-fallback-310.css
Correct reconciliation action on U+C entr intable from 'n' to 'c'

### CODE REVIEW

File: help/C/gnucash-help.xml [hunk 1]
```diff
@@ -343,7 +343,7 @@
       </author>,
       <author>
         <firstname>David</firstname> <surname>Cousens</surname>
-      </author>,
+      </author>
```
Comment (fellen) [inline help/C/gnucash-help.xml:346]: Why do you want to remove the comma? It is a long comma separated list of authors: https://code.gnucash.org/docs/C/gnucash-help/authors.html

**Comment by fellen** [comment]
> Correct external reference and link to gnucash-fallback-310.css

Where is it?

**Comment by Bob-IT** [comment]
> Correct external reference and link to gnucash-fallback-310.css

Can not see this in the commit but on master I have changed these file names, not realy sure they should be linked to, there is the example file in doc/gtk-3.0.css which I would of thought would be enough.

**Comment by fellen** [comment]
David, I have commited the typo fix in transactions.
For the other parts
Authors: intention unclear
css example (mentioned, but not part of this PR)
create one (or more, depending on the branch) separate PRs.

**Pull request closed without merging**